plugins/janus_audiobridge.c

```diff
@@ -233,6 +233,20 @@ room-<unique room ID>: {
 	"room" : <unique numeric ID of the destroyed room>
 }
 \endverbatim
+ *
+ * To enable or disable recording of mixed audio stream while the conference
+ * is in progress, you can make use of the \c enable_recording request,
+ * which has to be formatted as follows:
+ *
+\verbatim
+{
+	"request" : "enable_recording",
+	"room" : <unique numeric ID of the room>,
+	"secret" : "<room secret; mandatory if configured>"
+	"record" : <true|false, whether participants in this room should be automatically recorded or not>,
```

Comment:
I'd change this sentence to "whether this room should be automatically recorded or not", or otherwise someone reading the documentation might think this will record each participant individually to a separate .mjr (which the AudioBridge plugin does support as a feature). I guess this was copied from the VideoRoom docs, where that's indeed what's done.